src/test/scala/nl/knaw/dans/easy/solr/ConfSpec.scala

```diff
@@ -0,0 +1,57 @@
+package nl.knaw.dans.easy.solr
+
+import java.io.ByteArrayOutputStream
+
+import org.scalatest.{FlatSpec, Matchers}
+
+import scala.runtime.BoxedUnit
+
+class ConfSpec extends FlatSpec with Matchers {
+
+  "printHelp" should "produce help info with values from custom properties file" in {
+
+    val result = executeWithHomeDir("src/test/resources")
+
+    // assert values that are neither hard coded nor in assembly/dist but in test/resources
+    result should include("127.0.0.1")
+    result should include("(default = secret)")
+    result should include("(default = somebody)")
+    result should include("src/test/resources")
+    result should include("${project.version}") // FIXME ?
+  }
+
+  it should "be pasted into README.md" in {
```

Comment:
I am too lazy for the build-unzip-run cycle, or create run configurations in the IDE which I loose frequently. This way I don't need to remember anything. Now I can just run the build or the tests to get it. During development I frequently had second thoughts on the options and your review comment made me change it again. So I copy-pasted it also frequently.

Seeing it between the test results also reminds me I should copy-paste it!